modules/javafx.graphics/src/main/java/com/sun/javafx/sg/prism/NGPointLight.java

```diff
@@ -26,11 +26,85 @@
 package com.sun.javafx.sg.prism;
 
 /**
- * TODO: 3D - Need documentation
+ * The peer of the {@code PointLight} class. Holds the default values of {@code PointLight}'s
+ * properties and updates the visuals via {@link NGNode#visualsChanged} when one of the current
+ * values changes. The peer receives its changes by {@code PointLight.doUpdatePeer} calls.
  */
 public class NGPointLight extends NGLightBase {
 
+    /** Constant attenuation factor default value */
+    private static final double DEFAULT_CA = 1;
+    /** Linear attenuation factor default value */
+    private static final double DEFAULT_LA = 0;
+    /** Quadratic attenuation factor default value */
+    private static final double DEFAULT_QA = 0;
+    /** Max range default value */
+    private static final double DEFAULT_MAX_RANGE = Double.POSITIVE_INFINITY;
+
```

Comment:
If they are in `PointLight` it will be difficult for the peer to access them (will need to do through the accessor). Also, the parent `NGLightBase` holds the defaults for `color` and `lightOn`.